packages/amount-selectors/src/components/AmountSelectors.js

```diff
@@ -126,7 +126,6 @@ class AmountSelectors extends React.Component {
             onKeyDown={this.onKeyDown}
             onChange={this.onChange}
             onBlur={this.updateDisplayValue}
-            size={4}
```

Comment:
breaking change, should bump version number

Reply:
I'll rebase the commit message